### PR TITLE
new site and copyright

### PR DESCRIPTION
**Pull request by MaksimChepelev**

### CODE REVIEW

File: db/default.sql [hunk 1]
```diff
@@ -23,7 +23,7 @@ CREATE TABLE satellite (
   `updated_at` DATETIME NOT NULL
 );
 
-INSERT INTO satellite VALUES (2, 'ComSat4', 'comsat4.iotracks.com', '104.130.135.213', '','2016-09-21 15:21:31.898 +00:00',
```
Comment (ekrylovich) [inline db/default.sql]: not sure that here we should change something

File: package.json [hunk 2]
```diff
@@ -12,10 +12,10 @@
     "node"
   ],
   "author": "Zishan Iqbal",
-  "license": "Copyright (C) 2017 iotracks, inc. \nEclipse ioFog is provided under the Eclipse Public License (EPL) \nhttps://www.eclipse.org/legal/epl-v10.html",
+  "license": "Copyright (C) 2018 Edgeworx, Inc. \nEclipse ioFog is provided under the Eclipse Public License (EPL) \nhttps://www.eclipse.org/legal/epl-v10.html",
```
Comment (ekrylovich) [inline package.json]: ioFog???

Comment (MaksimChepelev) [reply package.json]: Edgeworks. I got it from Kilton

File: src/cli/Help.js [hunk 3]
```diff
@@ -104,7 +104,7 @@ function displayHelpFooter() {
   \tversion\t\t\t\t\t\t\t\tDisplays Version and License
   
 
-  \tReport bugs to: bugs@iotracks.com
+  \tReport bugs to: bugs@iofog.org
```
Comment (ilaryionava) [inline src/cli/Help.js]: did you get e-mail from kilton ?

File: src/cli/Help.js [hunk 4]
```diff
@@ -104,7 +104,7 @@ function displayHelpFooter() {
...
   \tioFog home page: http://iofog.com
```
Comment (ilaryionava) [inline src/cli/Help.js]: iofog.org ?

**Comment by ilaryionava** [comment]
I think there are more iotracks references in the code (resetPasswordTemp.js ,  emailActivationTemp.js, emailTemp.js & userController.js)

**Comment by ilaryionava** [comment]
all references of iotracks & iofabric, fabric, etc. should be replaced

**Pull request closed without merging**